ctapipe/image/muon/muon_ring_finder.py

```diff
@@ -1,28 +1,139 @@
 import numpy as np
 import astropy.units as u
-from ctapipe.image.muon.ring_fitter import RingFitter
+from ctapipe.core import Component
 from ctapipe.io.containers import MuonRingParameter
+import traitlets as traits
+from iminuit import Minuit
+
+__all__ = ['MuonRingFitter']
+
+
+class MuonRingFitter(Component):
```

Comment:
I think this should not contain the functions for taubin and chaudhuri fit. Define them as simple  functions outside the class that only get the inputs they need. Then call those functions here.

That makes it easier to test the single components and enhances readability.